// export PATH=$PATH:$HOME/go/bin
// protoc --go_out=plugins=grpc:proto proto/user.proto
// python3 -m grpc_tools.protoc -I. --python_out=. --grpc_python_out=. user.proto

syntax = "proto3";
package proto;
// option go_package = ".;user";

message User {
  int64 id = 1;
  string firstName = 2;
  string lastName = 3;
  string email = 4;
  string gender = 5;
  string ipAddress = 6;
  string userName = 7;
  string agent = 8;
  string country = 9;
}

message SaveResponse {
  string body = 1;
}

service UserService {
  rpc Save(User) returns (SaveResponse) {}
}